### PR TITLE
add CNI plugins to Vagrant install

### PR DESCRIPTION
**Pull request by tgross**
Needed this for testing of https://github.com/hashicorp/nomad/pull/6252

### CODE REVIEW

File: scripts/vagrant-linux-priv-cni.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+DOWNLOAD=https://github.com/containernetworking/plugins/releases/download/${VERSION}/cni-plugins-linux-amd64-${VERSION}.tgz
+
+function install_cni() {
+	mkdir -p /opt/bin/cni
+	if [[ -e /opt/bin/cni/${VERSION} ]] ; then
+		return
+	fi
+
+	wget -q -O /tmp/cni-plugins.tar.gz ${DOWNLOAD}
+	tar -xf /tmp/cni-plugins.tar.gz -C /opt/bin/cni
```
Comment (schmichael) [inline scripts/vagrant-linux-priv-cni.sh]: `-z` isn't needed?

Comment (tgross) [reply scripts/vagrant-linux-priv-cni.sh]: Not for extracting. https://www.xkcd.com/1168/ 😀

Comment (tgross) [reply scripts/vagrant-linux-priv-cni.sh]: Bah, but the paths are all wrong. Should be `/opt/cni/bin`.

Comment (tgross) [reply scripts/vagrant-linux-priv-cni.sh]: Fixed.

Comment (schmichael) [reply scripts/vagrant-linux-priv-cni.sh]: TIL!

Ugh good catch on the paths.

**Pull request merged**